src/spectrecoff-cli/commands/BreakdownChart.fs

```diff
@@ -33,36 +30,31 @@ type BreakdownChartExample() =
         ] |> toConsole
         0
 
+open SpectreCoff.Cli.Documentation
+
 type BreakdownChartDocumentation() =
     inherit Command<BreakdownChartSettings>()
     interface ICommandLimiter<BreakdownChartSettings>
 
     override _.Execute(_context, _settings) =
-        Theme.setDocumentationStyle
-        BL |> toConsole
-        pumped "BreakdownChart submodule"
-        |> alignedRule Left
-        |> toConsole
-
+        setDocumentationStyle
         Many [
-            C "This submodule provides functionality from the BreakdownChart widget of Spectre.Console ("
-            Link "https://spectreconsole.net/widgets/breakdownchart"
-            C ")"
+            docSynopsis "BreakdownChart submodule" "This submodule provides functionality from the BreakdownChart widget of Spectre.Console" "widgets/breakdownchart"
             BL
             C "The breakdown chart can be used using the breakdownChart function:"
             BI [
                 P "breakdownChart: ChartItem list -> OutputPayload"
             ]
             BL
-            Many [C "The"; P "ChartItem"; C "union type consists of two options:"]
+            C "The"; P "ChartItem"; C "union type consists of two options:"
             BI [
-                Many [P "ChartItem:"; C "Consists of the label and a value for the item."]
-                Many [P "ChartItemWithColor:"; C "Additionally defines a color the item will be rendered in."]
+                define "ChartItem" "Consists of the label and a value for the item."
```

Comment:
I was first a little sceptical about those utility methods, thinking they might add more confusion than is worth small loss of duplication. 
But on second thought, I think they are quite readable and make for more expressive strings (as does the output module in fact :))

Reply:
Well, thaey are only for the documentation, NOT for the examples above. But they do illustrate for users, how easily they can create composite functions for tempalting their output. Also, using this, we can actually easily change our documentation layout globally, by just tweaking functions, not having to revisit 20 files.